### PR TITLE
Adjust referral rewards

### PR DESCRIPTION
**Pull request by juliolrmonteiro**

### CODE REVIEW

**Comment by n13** [comment]
Looks good

File: src/seeds.accounts.cpp [hunk 1]
```diff
@@ -425,10 +426,24 @@ void accounts::update(name user, name type, string nickname, string image, strin
...
+  auto rem_balance = acc.balance;
+  if (quantity > rem_balance) {
+    // Should not fail in case not enough balance
+    // check(false, ("DEBUG: not enough balance on "+bankaccts::referrals.to_string()+" = "+rem_balance.to_string()+", required= "+quantity.to_string()).c_str());
+    return;
+  }
 
-  // TODO: Check balance - if the balance runs out, the rewards run out too.
+  // Checks the current SEEDS price from tlosto.seeds table
+  auto firstprice = pricehistory.rbegin()->seeds_usd;
```
Comment (n13) [inline src/seeds.accounts.cpp:440]: this is called "firstprice" but it is actually the last price entry, no?

File: src/seeds.accounts.cpp [hunk 2]
```diff
@@ -425,10 +426,24 @@ void accounts::update(name user, name type, string nickname, string image, strin
...
+  auto lastprice = pricehistory.begin()->seeds_usd;
```
Comment (n13) [inline src/seeds.accounts.cpp:441]: this is called lastprice but is actually the first price

**Review by n13 — CHANGES_REQUESTED** [review]
just name change of variable - everything else looks good

File: src/seeds.accounts.cpp (hunk 1 above)
Comment (juliolrmonteiro) [reply src/seeds.accounts.cpp]: Done, changed it to lastprice

File: src/seeds.accounts.cpp (hunk 2 above)
Comment (juliolrmonteiro) [reply src/seeds.accounts.cpp]: Done, changed it to initialprice

**Comment by juliolrmonteiro** [comment]
Also fixed all the tests, so that if "exchange.reset()" is uncommented and working the tests pass here.

**Review by n13 — APPROVED** [review]

**Review by n13 — APPROVED** [review]

**Pull request merged**